src/components/ProfilePage/components/AchievementsList/AchievementsList.tsx

```diff
@@ -12,8 +14,26 @@ export const AchievementsList = () => {
   const { data, fetchNextPage, hasNextPage, isFetchingNextPage } =
     useAchievementsInfinite(walletAddress);
 
+  const { t } = useTranslation();
+
   const pdas = data?.pages.flatMap((page) => page.data) || [];
 
+  if (!pdas.length) {
+    const entityTranslation = (
+      t('profile_page.achievements') || 'Achievements'
```

Comment:
Can we use the built-in default value? (https://www.i18next.com/translation-function/essentials#passing-a-default-value)


```suggestion
      t('profile_page.achievements', 'Achievements')
```

Reply:
Yes, yes, thank you for the close look 🙌